### DISCUSSION TITLE
Track name not displayed in monitor

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1012] [Help]
Currently the monitor doesnt display track name, which makes it almost unusable on large project.

I know Presonus use custom VST3 extensions to pass track name (as with Console 1 for example).

So it would be really nice and helpfull to add S1 support to display track name :)

More and more people are using Studio One 1 !

**Comment by apohl79** [comment]
I created a feature request for supporting the presonus VST3 extensions.
